platform/13-tekton-chains-setup.sh

```diff
@@ -31,4 +31,7 @@ kubectl patch \
       -n tekton-chains \
       --patch-file "$GIT_ROOT"/platform/components/tekton/chains/patch_ca_certs.json
 
+kubectl scale deployment -n tekton-chains tekton-chains-controller --replicas=0
+kubectl scale deployment -n tekton-chains tekton-chains-controller --replicas=1
```

Comment:
Would a `kubectl rollout restart` work? That would avoid overwriting a change to replicas if that's ever made elsewhere.

Reply:
Yup, just tested it and that works better. Thanks for the suggestion.